### ISSUE TITLE
The proposal to improve administrator's data explorer

### ISSUE THREAD

**Issue by kulakowka** [rethinkdb/rethinkdb#5506]
Could you implement a list of frequently used queries?

You can save in the history each executed query. But show each query only once. Last executed query should moves to the very top. Thus, we get a list of where the most frequent requests are at the top. And at the same time it will allow us to get rid of duplicate requests in history. Each request can be assigned a name and arrange them in the left sidebar. Or this may be similar to the list of url's in the Postman.

![2016-03-11 22 50 45](https://cloud.githubusercontent.com/assets/557190/13713772/bea57954-e7db-11e5-8a96-cbf8a68c0a70.png)

**Comment by danielmewes** [comment]
Thanks for the proposal @kulakowka .

This sounds very similar to https://github.com/rethinkdb/rethinkdb/issues/3928 . Do you mind if I merge this with the other issue?

**Comment by kulakowka** [comment]
@danielmewes yes, of course! Thank you for the link. I'll be watching.

**Comment by danielmewes** [comment]
Closing as duplicate of https://github.com/rethinkdb/rethinkdb/issues/3928
